submodules/linphone

```diff
@@ -1 +1 @@
-Subproject commit 73ee4c1f7f7eb2316e4ba81dd6642418e9f17757
+Subproject commit 44ae1322d2ec184d909c7b97167182d5fadbdffd
```

Comment:
Agreed. Submodules should match the SHA1 of our master. Do not merge until this is fixed.